### DISCUSSION TITLE
[Android] Disable Display Size Scaling

### DISCUSSION THREAD

**Discussion by Soap-141** [unoplatform/uno#13595] [Q&A]
Is it possible to disable the Android `Display size` setting? When default is not used it breaks the UI of our applications (most of the time).

![image](https://github.com/unoplatform/uno/assets/57812285/5aaaad03-7522-423f-bffc-5212a5bd6d80)

**Comment by jeromelaban** [comment]
Those two properties should be of help for this particular feature: 

https://github.com/unoplatform/uno/blob/669dbc7421905dcd8cb7caf09c56834d35ee24d9/src/Uno.UI/FeatureConfiguration.cs#L197-L203

**Reply by Soap-141** [reply]
Already using that, and it's not helping with this setting.

**Reply by jeromelaban** [reply]
Thanks. Then at this time we may not cover all cases. Can you open an issue?

**Reply by Soap-141** [reply]
Yes

**Reply by Soap-141** [reply]
Not sure it's doable, but https://github.com/unoplatform/uno/issues/13598 to leave a trace. I did my part 😝
